src/test/java/question/builders/FormAnsweredDtoBuilder.java

```diff
@@ -1,29 +1,24 @@
-package question.builders;
+package com.lumesse.soap.model.candidatewebservice;
+
 
 import dto.FormAnsweredDto;
 import dto.QuestionsAnsweredDto;
 
 public class FormAnsweredDtoBuilder {
-    //private List<QuestionAnsweredDto> questionAnswered = new ArrayList<>();
-    private FormAnsweredDto formAnsweredDto;
-    private QuestionsAnsweredDto questionsAnsweredDto;
+    private QuestionsAnsweredDto questionsAnswered = new QuestionsAnsweredDto();
 
     public static FormAnsweredDtoBuilder builder() {
         return new FormAnsweredDtoBuilder();
     }
 
     public FormAnsweredDtoBuilder withQuestionsAnswer(QuestionsAnsweredDto questionsAnsweredDto) {
-        formAnsweredDto.setQuestionsAnswered(questionsAnsweredDto);
+        questionsAnswered.getQuestionAnswered().addAll(questionsAnsweredDto);
```

Comment:
![image](https://user-images.githubusercontent.com/20994767/74602959-5b07f080-50ae-11ea-972f-79307605fa01.png)

Reply:
Use add not addAll - now You pass one element but collection is required.